index.js

```diff
@@ -4,6 +4,7 @@ const fs = require('fs');
 const execa = require('execa');
 const {isElectron} = require('electron-util/node');
 const macosVersion = require('macos-version');
+const screenCapturePermission = require('./build/Release/screencapturepermissions');
```

Comment:
I think I would leave the require guarded below in the function so that its only loaded if running on mac. Then the module itself can be imported also on linux and windows without causing issues. 

Previously it looked like this: https://github.com/saghul/mac-screen-capture-permissions/blob/652e10c1482d2903007edfc31e97324402dca7ec/index.js#L25

Reply:
Makes sense, I'll make the change. Thanks for taking a look!